### DISCUSSION TITLE
Làm bài từn 11 thui nè

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#15] [General]
Câu hỏi tìm hiểu nè:
  Bài 9: Tin học và xã hội
Câu 1:Hãy nêu sự ứng dụng tin học của bản thân cho việc học, làm việc trong cuộc sống (nếu làm nhóm, mỗi thành viên nêu ra).
Câu 2:Trong thời đại công nghệ phát triển, cập nhật thay đổi liên tục; liệu những xu hướng sử dụng công nghệ hiện tại có phù hợp tất cả với suy nghĩ, sự vận hành của xã hội? Hãy nêu 5 ví dụ và từ đó nói ra suy nghĩ bản thân, có nên giữ nguyên hay thay đổi chúng?
Câu 3:Thông tin của bản thân có quan trọng không? Hãy “vẽ” ra những tình huống thông tin cá nhân bị kẻ xấu sử dụng và cách khắc phục, bảo vệ.
  Bài 10: Khái niệm về hệ điều hành
Câu 1:Hệ điều hành là gì? Phân biệt các loại hệ điều hành
Câu 2:Hệ điều hành thường được lưu trữ ở đâu? Vì sao phải nạp hệ điều hành vào máy tính thì máy mới hoạt động được?

**Comment by HuynhLeBaoTran** [comment]
alo alo

**Comment by LTGHan** [comment]
Yo

**Comment by HuynhLeBaoTran** [comment]
hí looo

**Comment by GiaHan06** [comment]
Câu 1 bài 9 mỗi bạn 1 ý nha

**Reply by chaudambaotram** [reply]
vậy là câu 1 tất cả thành viên đều tham gia hẻ?

**Reply by GiaHan06** [reply]
đúng rùi, mũi ngừi 1 ý thoi hà

**Comment by HuynhLeBaoTran** [comment]
z thì 4 câu chia ntn

**Reply by chaudambaotram** [reply]
tui nghĩ câu 2 B9 nên cho 2 người làm í vì co VD nè

**Reply by GiaHan06** [reply]
đúng rùi á tui đang định nhắn

**Reply by HuynhLeBaoTran** [reply]
okk

**Reply by chaudambaotram** [reply]
Vậy cho 2 người i

**Reply by GiaHan06** [reply]
nhma sy nghĩ mỗi ng có thể là khác nhau á nên chắc để 2 ý luôn cũng được hoặc làm chung,...

**Comment by LTGHan** [comment]
Tui câu 3 bài 9 nha

**Comment by HuynhLeBaoTran** [comment]
z tui c1 b10 nha

**Comment by GiaHan06** [comment]
@NguyenNgocQuynhAnh  đou ròi bà

**Comment by NguyenNgocQuynhAnh** [comment]
còn câu nào z mấy bà

**Comment by NguyenNgocQuynhAnh** [comment]
c2 b10 có ai làm chưa za

**Reply by HuynhLeBaoTran** [reply]
chưa nè

**Reply by GiaHan06** [reply]
chưa á

**Reply by GiaHan06** [reply]
@NguyenNgocQuynhAnh zị bà làm c2 10 hã

**Reply by NguyenNgocQuynhAnh** [reply]
oke tui làm he

**Comment by GiaHan06** [comment]
@chaudambaotram  ơi, làm c2 bài 9 zới tui nha bà

**Reply by chaudambaotram** [reply]
ok bà nha
